ros2_batch_job/__main__.py

```diff
@@ -483,11 +484,13 @@ def run(args, build_function, blacklisted_package_names=None):
         if sys.platform == 'win32':
             if args.cmake_build_type == 'Debug':
                 pip_packages += [
+                    # TODO cryptography,
```

Comment:
```suggestion
                    'https://github.com/ros2/ros2/releases/download/cryptography-archives/cffi-1.12.3-cp37-cp37dm-win_amd64.whl',  # required by cryptography
                    'https://github.com/ros2/ros2/releases/download/cryptography-archives/cryptography-2.7-cp37-cp37dm-win_amd64.whl',
```

Reply:
Thanks, committed to my branch but @jacobperron I'll need you to move merge it to the ros2 org branch for us to test it

Reply:
Right, forgot about that. Will do.

Reply:
@emersonknapp You've added the `cffi` wheel (which the cryptography wheel needs), but you should also add the cryptography wheel (see [my branch](https://github.com/ros2/ci/blob/2ca84dede91d625812c5ceacae3283604d14bed9/ros2_batch_job/__main__.py#L487-L488)). Anyways, I've triggered the build :crossed_fingers:

Reply:
Oh - sorry I'll update that - manually copied it over to get the CDO - missed that line